src/core/cubic.c

```diff
@@ -311,6 +372,7 @@ CubicCongestionControlOnCongestionEvent(
         Cubic->KCubic = S_TO_MS(Cubic->KCubic);
         Cubic->KCubic >>= 3;
 
+        CubicCongestionHyStartChangeState(Cc, HYSTART_DONE);
```

Comment:
Feels weird that this is excluded from the persistent congestion case, and also I feel like a fast path for when the connection is already in HYSTART_DONE might make sense here

Reply:
What do you mean by "fast path"?

Reply:
I mean don't do all that stuff in CubicCongestionHyStartChangeState if we are already in HYSTART_DONE